OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -345,11 +345,14 @@ public IHardwareCursor CreateHardwareCursor(string name, Size size, byte[] data,
 					hotspot *= 2;
 				}
 
-				return new Sdl2HardwareCursor(size, data, hotspot);
+				var cursor = new Sdl2HardwareCursor(size, data, hotspot);
+				return cursor?.Cursor == IntPtr.Zero ? null : cursor;
 			}
 			catch (Exception ex)
 			{
-				throw new Sdl2HardwareCursorException($"Failed to create hardware cursor `{name}` - {ex.Message}", ex);
+				Log.Write("debug", "Failed to create hardware cursor `{0}` - {1}", name, ex.Message);
```

Comment:
Was changing from string interpolation back to older style format args intentional?

Reply:
This was adapted from the lines in `CursorManager`. I admit I noticed this but didn't try to use interpolation since I thought there was a reason we kept `CursorManager` like that.

Reply:
> since I thought there was a reason we kept CursorManager like that.

I wouldn't be surprised if we simply missed some cases, or they kept sneaking back in through older PRs that didn't show rebase conflicts.